cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -163,36 +163,51 @@ def test_provision_approval(appliance, provider, vm_name, smtp_test, request,
 
     # It will provision two of them
     vm_names = [vm_name + "001", vm_name + "002"]
-    requester = "" if BZ(1628240, forced_streams=['5.10']).blocks else "vm_provision@cfmeqe.com "
+    if BZ(1628240).blocks and provider.one_of(CloudProvider):
```

Comment:
Why is this made provider specific?

Reply:
because infra providers have requester in this message whereas cloud providers don't have it.